frontend/src/setupTests.js

```diff
@@ -20,3 +20,9 @@ import { configure } from "enzyme"
 import Adapter from "@wojtekmaj/enzyme-adapter-react-17"
 
 configure({ adapter: new Adapter() })
+
+/* eslint @typescript-eslint/no-var-requires: "off" */
+// needed for BokehChart.test.tsx as we import bokeh scripts that use message channel
+// https://github.com/jsdom/jsdom/issues/2448
+// need to add --experimental-worker in node options craco test cli command
```

Comment:
This comment would also benefit from a clarity pass. Generally, sentence fragments and no punctuation isn't an issue in comments. But in this case, these are, I assume, two _separate_ ideas ("We need to add the window.MessageChannel import here because..." and "We need to also add this --experimental-worker param elsewhere because"). 

Can you:
- Add some punctuation (or an additional line break or whatever) so it's clear that these are two separate things being communicated
- Rework that second point "need to add --experimental-worker". Who needs to add it? (Presumably you've already added it, so it's more "Additionally, we also modified such-and-such file to add such and such params" - it's not an imperative statement that the reader needs to act on.)
- "node options craco test cli command" sounds like a series of disconnected words. My assumption is, you edited a _specific file_ with that _specific param_ in order to get bokeh tests to work. Can this comment instead say something like "Additionally: the `test` command in `package.json` uses the `--experimental-worker` node option, which is required to make Bokeh tests work."